content/source/docs/glossary.html.md

```diff
@@ -346,6 +354,12 @@ In Terraform Cloud, most internal application objects (like [workspaces][], user
 
 You can usually copy an external ID from the URL bar when viewing an object in Terraform Cloud's UI. Workspaces don't display an ID in the URL bar, but their general settings page includes a UI control for viewing and copying the ID.
 
+## Infrastructure as Code
+
+[infrastructure-as-code]: glossary.html#infrastructure-as-code
+
+Infrastructure-as-Code (IaC) is the process of managing infrastructure in a file or files rather than manually configuring it via a user interface. Infrastructure resource types managed with IaC can include virtual machines, security groups, network interfaces, and many others.
```

Comment:
This feels more like an ongoing circular "practice" than a beginning/middle/end "process"... y/n?

```suggestion
Infrastructure-as-Code (IaC) is the practice of managing infrastructure in a file or files rather than manually configuring it via a user interface. Infrastructure resource types managed with IaC can include virtual machines, security groups, network interfaces, and many others.
```